### PR TITLE
Workshops & Seminars data

### LINKED ISSUE #157: Add 2020 workshops data

### PR DESCRIPTION
**Pull request by AliLordLoss**
closes #157

### CODE REVIEW

File: backend/people/migrations/0017_fill_more_speakers_data.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from django.db import migrations
+
+def forwards(apps, schema_editor):
+    Speakers = apps.get_model('people', 'Speaker')
+    Speakers.objects.create(
+        name='Mostafa Rezazad',
+        degree='Ph.D.',
+        place='senior researcher at the Institute for Research in Fundamental Sciences',
+        bio='Dr Mostafa Rezazad Received his Ph.D. from the National University of Singapore. Then he joined the CiSDeM group at SUTD university to work on applying AI in the Cybersecurity domain. Now, he is a senior researcher at the Institute for Research in Fundamental Sciences where he is pursuing his research in various directions in the Networking field.'
+    )
```
Comment (ahmadsalimi) [inline backend/people/migrations/0017_fill_more_speakers_data.py:13]: Please fill the `polymorphic_ctype` field of all polymorphic models, such as Speaker, Staff, etc.
Like this:
```Python
from django.contrib.contenttypes.models import ContentType

Speaker = apps.get_model('people', 'Speaker')
ctype = ContentType.objects.get_for_model(Speaker)

Speaker.objects.create(
    name='Mostafa Rezazad',
    degree='Ph.D.',
    place='senior researcher at the Institute for Research in Fundamental Sciences',
    bio='Dr Mostafa Rezazad Received his Ph.D. from the National University of Singapore. Then he joined the CiSDeM group at SUTD university to work on applying AI in the Cybersecurity domain. Now, he is a senior researcher at the Institute for Research in Fundamental Sciences where he is pursuing his research in various directions in the Networking field.',
    polymorphic_ctype=ctype
)
```

File: backend/people/migrations/0018_update_holding_team_orders.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        'Content',
+        'Technical',
+        'Network',
+        'Branding',
+        'Social',
+        'Media',
+        'Presentation Management'
+    ]
+
+    HoldingTeam.objects.get(wss=wss, name='Event Director', order=1).update(name='Manager & Director')
```
Comment (ahmadsalimi) [inline backend/people/migrations/0018_update_holding_team_orders.py:21]: I think filtering on the `order` field is not necessary.

File: backend/people/migrations/0018_update_holding_team_orders.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    Staff.objects.create(
+        name='Shayan Oveis gharan',
+        picture='media/2020/staff/Shayan-Oveis-gharan.jpg'
+    )
+
```
Comment (ahmadsalimi) [inline backend/people/migrations/0018_update_holding_team_orders.py:27]: Set `polymorphic_ctype` field for this object.

File: backend/people/migrations/0018_update_holding_team_orders.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    scientific = HoldingTeam.objects.create(wss=wss, name='Scientific Secretary', order=2)
+    scientific.staff.add(Staff.objects.get(name='Shayan Oveis gharan'))
```
Comment (ahmadsalimi) [inline backend/people/migrations/0018_update_holding_team_orders.py:29]: I'm not sure but I think you need to call the `scientific.save()` method to save the changes. Please check the presence of the object after running locally.

File: backend/people/migrations/0019_fill_more_seminars.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        speaker=Speaker.objects.filter(name='Mostafa Rezazad').last()
+    )
```
Comment (ahmadsalimi) [inline backend/people/migrations/0019_fill_more_seminars.py:19]: Please set the `polymorphic_ctype` for these objects.

File: backend/people/migrations/0020_fill_workshops.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
```
Comment (ahmadsalimi) [inline backend/people/migrations/0020_fill_workshops.py:5]: The workshop model is in the `events` app. So, this migration must be in the events app.

File: backend/people/migrations/0020_fill_workshops.py [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    wss = WSS.objects.get(year=2020)
+
+    Workshop.objects.create(
+        wss=wss,
+        title='Conceptual Blending in Computational Creativity',
+        syllabus='Modeling humans\' cognitive processes in a computable way is our goal. While machines communicate with humans, they need to understand complicated products produced by human’s creativity. Human creativity appears in music, art, jokes, novel stories, movies, and every other area. My current focus is on conceptual blending. Conceptual blending (CB) is a vital cognitive mechanism by which two or more mental spaces are integrated to produce new concepts (Fauconnier and Turner 2002), e.g., when a house and boat blend and a house-bout is created. Visual Metaphor is a similar area. Despite not all visual blends being visual metaphors, most research is conducted about advertisements and focuses on them. Creative systems can understand human creativity, produce innovative products, and enhance human creativity (The International Association for Computational Creativity (ACC)). I wish to explain this fantastic area in detail.',
+        speaker=Speaker.objects.filter(name='Maryam Sadat Eslami').last()
+    )
```
Comment (ahmadsalimi) [inline backend/people/migrations/0020_fill_workshops.py:19]: Please set the `polymorphic_ctype` for these objects.

File: backend/people/migrations/0019_fill_more_seminars.py [hunk 8]
```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
```
Comment (ahmadsalimi) [inline backend/people/migrations/0019_fill_more_seminars.py:5]: The seminar model is in the events app. So, this migration must be in the events app.

**Review by ahmadsalimi — DISMISSED** [review]

File: backend/people/migrations/0017_fill_more_speakers_data.py (hunk 1 above)
Comment (AliLordLoss) [reply backend/people/migrations/0017_fill_more_speakers_data.py:13]: Done

File: backend/people/migrations/0018_update_holding_team_orders.py (hunk 2 above)
Comment (AliLordLoss) [reply backend/people/migrations/0018_update_holding_team_orders.py:21]: Done

File: backend/people/migrations/0018_update_holding_team_orders.py (hunk 3 above)
Comment (AliLordLoss) [reply backend/people/migrations/0018_update_holding_team_orders.py:27]: Done

File: backend/people/migrations/0018_update_holding_team_orders.py (hunk 4 above)
Comment (AliLordLoss) [reply backend/people/migrations/0018_update_holding_team_orders.py:29]: check migration number 13 line 25 :)))
I'll check in local run too

File: backend/people/migrations/0019_fill_more_seminars.py (hunk 5 above)
Comment (AliLordLoss) [reply backend/people/migrations/0019_fill_more_seminars.py:19]: check migration number 11
do I need to set this for them too?

File: backend/people/migrations/0019_fill_more_seminars.py (hunk 8 above)
Comment (AliLordLoss) [reply backend/people/migrations/0019_fill_more_seminars.py:5]: Done

File: backend/people/migrations/0020_fill_workshops.py (hunk 7 above)
Comment (AliLordLoss) [reply backend/people/migrations/0020_fill_workshops.py:19]: Done

File: backend/people/migrations/0020_fill_workshops.py (hunk 6 above)
Comment (AliLordLoss) [reply backend/people/migrations/0020_fill_workshops.py:5]: Done

File: backend/events/migrations/0008_fill_more_seminars.py [hunk 9]
```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
```
Comment (emranbm) [inline backend/events/migrations/0008_fill_more_seminars.py]: rename file to `0008_fill_more_seminars_of_2020.py`

File: backend/events/migrations/0009_fill_workshops.py [hunk 10]
```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
```
Comment (emranbm) [inline backend/events/migrations/0009_fill_workshops.py]: And here `fill_workshops_2020`

**Review by emranbm — APPROVED** [review]

**Review by emranbm — APPROVED** [review]

**Pull request merged**